docs/webui.md

```diff
@@ -268,33 +306,31 @@ Clicking on the action icon <Icon icon='mdi:delete-outline' fontSize='24' color=
 
 Displays a confirmation dialog, clicking on the SUBMIT button deletes this catalog.
 
-![delete-catalog](assets/webui/delete-catalog.png)
+![delete-catalog](./assets/webui/delete-catalog.png)
 
 ### Schema
 
-Under Construction...
+Design draft preview:
+
```

Comment:
I think you don't have to mention "Design draft preview", you can delete it here and below.